pages/agent/clusters.md

```diff
@@ -0,0 +1,33 @@
+# Clusters
+
+Clusters are a new way of managing your Buildkite agents. They allow teams to self-manage their agent pools, let admins create isolated sets of agents and pipelines within the one Buildkite organization, and help to make agents and queues more discoverable across your organization.
+
+You can turn on clusters for your organization if you have an administrator role through Organization Settings >> Pipeline Settings. Once you turn it on, you won’t be able to turn it back off.
+
+## What's changed?
+
+* Clusters are viewable to your entire organization, allowing engineers to understand better what agents and queues are available in their pipelines.
+* You can grant cluster-admin access to individual users or teams to allow them to self-administer their own sets of agents. Cluster admins can create and delete queues, access agent tokens, and add and remove pipelines.
+* Queues can be created within a cluster and validated when a pipeline's configuration is saved during pipeline upload, and when a new agent connects. This helps to make queues more discoverable to pipeline authors and helps to prevent ghost jobs that never run because they had a typo in their pipeline.yml's queue name.
```

Comment:
>  because they had a typo in their pipeline.yml's queue name.

I don't think we need this bit

Reply:
Keeping this benefit in place but rephrasing slightly sounds good to me 👍🏼